LSP.sublime-settings

```diff
@@ -93,6 +93,7 @@
   // Show code actions:
   // "annotation" - show an annotation on the right when code actions are available.
   // "bulb" - show a bulb in the gutter when code actions are available.
+  // "" - don't show code actions.
```

Comment:
Would prefer `false` to be honest but I guess this simplifies types?

Reply:
yes, what would `true` mean then?

Reply:
There doesn't need to be "true".
Or it can default to whatever is the default.

Reply:
Maybe `"none"` as it is used in various ST preferences.

Reply:
We already use `""` for diagnostics_highlight_style, document_highlight_style, diagnostics_gutter_marker.

Reply:
Right, and "none" would probably only fit here if the setting was used to show only a subset of code actions or so. I guess empty string is fine.